cds/modules/deposit/static/templates/cds_deposit/types/project/actions.html

```diff
@@ -2,7 +2,7 @@
   ng-hide='$ctrl.cdsDepositCtrl.record._deposit.status == "published" || $ctrl.cdsDepositCtrl.isPristine()'
   ng-disabled="$ctrl.cdsDepositCtrl.cdsDepositsCtrl.loading"
   class="btn btn-sm btn-success"
-  ng-click="$ctrl.saveAllPartial()">
+  ng-click="$ctrl.actionHandler('SAVE_PARTIAL')">
```

Comment:
Are you sure is not `saveAllPartial` here?

Reply:
nope it wasn't :P